net/mwan3/files/usr/sbin/mwan3track

```diff
@@ -37,6 +37,7 @@ main() {
 	trap if_down SIGUSR1
 
 	config_load mwan3
+	config_get track_method $1 track_method ping
```

Comment:
@marcin1j 
my be at this position a check with "which arping &>/dev/null", "which httping &>/dev/null" if 0 use it and otherwise log warning and use ping.